web_no_crawler/controllers/main.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2018 Xpansa Group
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import http
+
+
+class Main(http.Controller):
+
+    @http.route('/robots.txt', type='http', auth="none")
+    def robots_txt(self):
```

Comment:
in website v8 this method is named "robots" https://github.com/odoo/odoo/blob/8.0/addons/website/controllers/main.py#L86